tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp

```diff
@@ -520,22 +526,20 @@ TEST(TfGeometry, Quaternion)
     q1.header.frame_id = "A";
 
     // simple api
-    const geometry_msgs::msg::QuaternionStamped q_simple = tf_buffer->transform(
-      q1, "B", tf2::durationFromSec(
-        2.0));
-    EXPECT_NEAR(q_simple.quaternion.x, M_SQRT1_2, EPS);
-    EXPECT_NEAR(q_simple.quaternion.y, 0, EPS);
-    EXPECT_NEAR(q_simple.quaternion.z, -1 * M_SQRT1_2, EPS);
-    EXPECT_NEAR(q_simple.quaternion.w, 0, EPS);
+    {
+      const geometry_msgs::msg::QuaternionStamped q_simple = tf_buffer->transform(
+        q1, "B", tf2::durationFromSec(
+          2.0));
+      EXPECT_PRED3(CheckQuaternionNear, q_simple.quaternion, tf2::Quaternion(M_SQRT1_2, 0, -M_SQRT1_2, 0), EPS);
```

Comment:
```suggestion
      EXPECT_PRED3(CheckQuaternionNear, q_simple.quaternion,
        tf2::Quaternion(M_SQRT1_2, 0, -M_SQRT1_2, 0), EPS);
```
Address linter concerns: `tf2_geometry_msgs.cpplint.whitespace/line_length [2] (/tmp/ws/src/geometry2/tf2_geometry_msgs/test/test_tf2_geometry_msgs.cpp:533)`